libsolidity/formal/SymbolicTypes.cpp

```diff
@@ -139,7 +139,22 @@ pair<bool, shared_ptr<SymbolicVariable>> newSymbolicVariable(
 	else if (isBool(_type.category()))
 		var = make_shared<SymbolicBoolVariable>(type, _uniqueName, _context);
 	else if (isFunction(_type.category()))
-		var = make_shared<SymbolicFunctionVariable>(type, _uniqueName, _context);
+	{
+		auto const& fType = dynamic_cast<FunctionType const*>(type);
+		auto params = fType->parameterTypes() + fType->returnParameterTypes();
+		auto const& functionParam = find_if(
+			begin(params),
+			end(params),
+			[&](TypePointer _paramType) { return _paramType->category() == solidity::Type::Category::Function; }
+		);
+		if (functionParam != end(params))
+		{
+			abstract = true;
+			var = make_shared<SymbolicIntVariable>(TypeProvider::uint256(), type, _uniqueName, _context);
```

Comment:
Functions that have function pointers as arguments cannot be declared, so those we abstract fully.